modules/two_factor_authentication/app/models/two_factor_authentication/device/totp.rb

```diff
@@ -24,7 +24,7 @@ def self.supported_channels
     ##
     # verify the given OTP input
     def verify_token(token)
-      result = totp.verify_with_drift_and_prior(token.to_s, allowed_drift, last_used_at)
+      result = totp.verify(token.to_s, drift_behind: allowed_drift, drift_ahead: allowed_drift, after: last_used_at)
```

Comment:
The change should be equivalent to what we had before. Can you please verify, @oliverguenther :

https://github.com/mdp/rotp/blob/v3.x/lib/rotp/totp.rb#L44

Reply:
looks correct to me